app/_hub/kong-inc/graphql-rate-limiting-advanced/index.md

```diff
@@ -150,18 +161,20 @@ params:
       default: sliding
       value_in_examples:
       description: |
-        This sets the time window to either `sliding` or `fixed`
+        This sets the time window to either `sliding` or `fixed`.
   extra: |
     > Note:  Redis configuration values are ignored if the `cluster` strategy is used.
```

Comment:
Shouldn't this note also be part of the list below?

Reply:
> Shouldn't this note also be part of the list below?

It was my impression that it is a separate note directly under the table and so I did not more it.